### PR TITLE
SAR-7384  Create view and controller for scenario 2

### PR DESCRIPTION
**Pull request by EviNikolaidou**

### CODE REVIEW

File: app/uk/gov/hmrc/claimvatenrolmentfrontend/controllers/UnmatchedUserErrorController.scala [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+import javax.inject.{Inject, Singleton}
+
+import scala.concurrent.{ExecutionContext, Future}
+
+@Singleton
+class UnmatchedUserErrorController @Inject()(mcc: MessagesControllerComponents,
+                                             view: unmatched_user_error_page,
+                                             val authConnector: AuthConnector
+                                               )(implicit appConfig: AppConfig, ec: ExecutionContext) extends FrontendController(mcc) with AuthorisedFunctions {
+
```
Comment (HarrNM) [inline app/uk/gov/hmrc/claimvatenrolmentfrontend/controllers/UnmatchedUserErrorController.scala:33]: This doesn't need the AuthorisedFunctions

File: it/uk/gov/hmrc/claimvatenrolmentfrontend/controllers/UnmatchedUserErrorControllerISpec.scala [hunk 2]
```diff
@@ -0,0 +1,22 @@
+
+package uk.gov.hmrc.claimvatenrolmentfrontend.controllers
+
+import play.api.test.Helpers._
+import uk.gov.hmrc.claimvatenrolmentfrontend.assets.TestConstants.{testInternalId, testJourneyId}
```
Comment (HarrNM) [inline it/uk/gov/hmrc/claimvatenrolmentfrontend/controllers/UnmatchedUserErrorControllerISpec.scala:5]: Unused imports

File: app/uk/gov/hmrc/claimvatenrolmentfrontend/controllers/UnmatchedUserErrorController.scala (hunk 1 above)
Comment (r-melvin) [reply app/uk/gov/hmrc/claimvatenrolmentfrontend/controllers/UnmatchedUserErrorController.scala:31]: Can you move this controller into the error pages folder please?

File: conf/messages [hunk 3]
```diff
@@ -90,4 +90,9 @@ known_facts_mismatch.line_1                                 = The information yo
...
 invalid_account_type.title                                  = You cannot use this service
 invalid_account_type.heading                                = You cannot use this service
-invalid_account_type.line_1                                 = Contact the person who set up your account.
\ No newline at end of file
+invalid_account_type.line_1                                 = Contact the person who set up your account.
+
+# Unmatched User Error Page
+unmatched_user.title                                        = You cannot use this service
+unmatched_user.heading                                      = You cannot use this service
+unmatched_user.line_1                                       = Go to your business tax account to <a id="businessTaxAccount" href="{0}" target="_blank">manage who can access your taxes, duties and schemes</a>.
```
Comment (r-melvin) [inline conf/messages:98]: Is there a way we can do this without putting html in the messages file?

**Pull request merged**